doc/toolchain.rst

```diff
@@ -52,6 +52,11 @@ control the compilation process:
     ``HILTI_CXX``
         Specifies the path to the C++ compiler to use.
 
+    ``HILTI_CXX_COMPILER_LAUNCHER``
```

Comment:
Can we extend this change to automatically use `ccache` when `configure` found one?

Reply:
I added an extra patch for that automates this, but still allows opt-out.